Assigment2/assignment2/Cricle.java

```diff
@@ -0,0 +1,20 @@
+package assignment2;
+public class Cricle {
+    
+    public void calculation() {
+        int radius = 5;
+        try {
+
+            double area, perimeter;
+            area = 3.14 * radius * radius;
+            perimeter = 2 * 3.14 * radius;
+
+            System.out.println("Area of Circle: " + area);
+            System.out.println("Perimeter of Circle: " + perimeter);
+        } catch (Exception e) {
+            System.out.println("Error : " + e);
```

Comment:
Can you please throw an exception from this method?

Reply:
> Can you please throw an exception from this method?

yes sir . 
I did it .